arangodb-net-standard/IndexApi/Models/PostMultiDimensionalIndexBody.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace ArangoDBNetStandard.IndexApi.Models
+{
+    /// <summary>
+    /// Request body to create a Multi-Dimensional index
+    /// </summary>
+    public class PostMultiDimensionalIndexBody : PostIndexBody
+    {
+        public PostMultiDimensionalIndexBody()
+        {
+            Type = IndexTypes.MultiDimensional;
+            FieldValueTypes = "double";
+        }
+
+        /// <summary>
+        /// Required. Always initialized to the value "double". 
+        /// Currently only doubles are supported as values.
+        /// </summary>
+        public string FieldValueTypes { get; private set; }
```

Comment:
> Currently only doubles are supported as values.

Minor: What if this changes in future? We would have to change `private set` to `set`. Maybe we should just do it now? ArangoDB will return an error if something else is provided.

Reply:
Changed `private set` to `public set`